content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -37,6 +37,15 @@ To get started, check out https://istio.io/latest/docs/setup/getting-started/
 
 {{< boilerplate helm-backup >}}
 
+### Upgrading Custom Resource Definitions (CRDs)
+
+{{< warning >}}
+[Helm does not upgrade or delete CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations) when performing an upgrade. Because of this restriction, an additional step is required when upgrading Istio with Helm.
+{{< /warning >}}
+
+Istio ships CRDs with its helm charts; they are not templated and are found in the base chart under
+the `crds` folder. Upgrading Istio with Helm requires a change in workflow due to the required updating of CRDs for proper functionality.
```

Comment:
There's no action in this section.

These two sentences still say the same thing as each other.
How about removing the second one, and making this one sentence without a warning?

Alternatively, how about moving the warning up to the "Upgrade steps" section and removing this section entirely?

Reply:
I went with the latter.